src/interpreter/vm/DurianObject.cpp

```diff
@@ -0,0 +1,31 @@
+#include <DurianObject.h>
+#include <ostream>
+
+DurianObject::DurianObject() {
```

Comment:
Personally I prefer [member initializers](http://en.cppreference.com/w/cpp/language/initializer_list). That link is a bit confusing but it's what I used in [`Lexer`](https://github.com/ubclaunchpad/durian/blob/master/src/lexer/Lexer.cpp#L4).